kurtosis-devnet/pkg/kurtosis/sources/deployer/wallets.go

```diff
@@ -0,0 +1,106 @@
+package deployer
+
+// TODO: we should be getting this from somewhere in the enclave. It's ok-ish
+// though, as this is hard-coded in ethereum-optimism anyway.
+var knownWallets = []*Wallet{
+	{
```

Comment:
There's a `op-chain-ops/devkeys` package that may be useful for dev purposes, where keys are set up all from the same mnemonic, in a structured way, for all chains / roles.

Reply:
ah cool. Will take a look

Reply:
Actually, do you mind if I do that in a followup PR?
I figured I should probably use the same for  the operators keys, but I need to check I'm getting/using the right mnemonics/roles/... which will be easier once I land a workable end-to-end tool

Reply:
Nevermind, I started using devkeys for the users.
We'll  see later what other keys are useful